CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 11.14
+Update content-api-models to 11.14:
+* Add OphanStoryQuestionsResponse
```

Comment:
Can we omit this line please? I never want OphanStoryQuestionsResponse to be known by or used by the scala client as it will eventually disappear and we make no guarantees about it.